TackleTabby/Assets/Scripts/UI/ToggleHighlight.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UIElements;
+public class ToggleHighlight : MonoBehaviour , IOverlayMenu
+{
+    [SerializeField]
+    private Canvas CatalogueCanvas;
+    [SerializeField] 
+    private Canvas HungerBarCanvas;
+
+    [SerializeField] 
+    private GameObject FieldGrayout;
+    [SerializeField] 
+    private GameObject CatalogueHighlight;
+    [SerializeField]
+    private GameObject HungerBarHighlight;
+    
+    [SerializeField]
+    private float waitingTime = 0.5f;
```

Comment:
Please keep to the naming convention:

```suggestion
    private float WaitingTime = 0.5f;
```